rest_framework/fields.py

```diff
@@ -873,20 +875,28 @@ def to_internal_value(self, data):
 
 # Number types...
 
-class IntegerField(Field):
+
+class NumberField(Field):
```

Comment:
Not sure I clearly understand what you mean... You'll prefer that I just patch the FloatField (which is the starting point of the localization problem) to work like the DecimalField with possible code duplication than using a new base class? I agree that will have a lower impact and it could be a less "risky" change. Do I understand right?